platforms/quarkus/ws/src/main/resources/application.properties

```diff
@@ -9,6 +9,7 @@
 %dev.quarkus.datasource.jdbc.min-size=2
 
 
+%dev.quarkus.http.port=${HTTP_PORT:8080}
```

Comment:
Should we default this one to another port?  The API is also defaulted to 8080.

Reply:
+1

Reply:
Same here, I don't think this is adding something new.